.github/workflows/serverless-integration.yml

```diff
@@ -23,8 +23,8 @@ jobs:
         with:
           node-version: 14
       
-      - name: Install Serverless Framework
-        run: sudo yarn global add serverless --prefix /usr/local
+      - name: Install Serverless Framework 
+        run: sudo yarn global add serverless@^2.72.2 --prefix /usr/local
```

Comment:
Serverless Framework 3 was just released and includes breaking changes. Without this, GitHub will install the latest version when running the tests. We need to stay on version 2 for now.